app/models/miq_server/worker_management/heartbeat.rb

```diff
@@ -91,11 +91,23 @@ def validate_heartbeat(w)
     end
   end
 
+  def clean_heartbeat_files
+    Dir.glob(Rails.root.join("tmp", "*.hb")).each { |f| File.delete(f) }
+  end
+
   private
 
-  def workers_last_heartbeat(pid)
+  def workers_last_heartbeat(w)
+    ENV["WORKER_HEARTBEAT_METHOD"] == "file" ? workers_last_heartbeat_to_file(w) : workers_last_heartbeat_to_drb(w)
```

Comment:
@gtanzillo This is pedantic, but while you are addressing @jrafanie 's [comment](https://github.com/ManageIQ/manageiq/pull/15377/files#r123808310), I feel like these methods should use `from` instead of `to`:

- `workers_last_heartbeat_to_file => workers_last_heartbeat_from_file`
- `workers_last_heartbeat_to_drb => workers_last_heartbeat_from_drb `